themes/theme-gmd/components/ProductCard/index.jsx

```diff
@@ -56,6 +58,7 @@ const ProductCard = ({
             <Ellipsis rows={titleRows || 3}>{product.name}</Ellipsis>
           </div>
         )}
+        <MapPriceHint productId={product.id} location="slider" />
         {!hidePrice && (
           <Fragment>
             <Portal name={portals.PRODUCT_ITEM_PRICE_BEFORE} props={{ productId: product.id, location: 'productCard' }} />
```

Comment:
Please don't define these objects inline. Define them before the react output and reuse the variable instead. This is performance and prevention of unnecessary renderings.